src/loaders/compute_tools/tool_common.py

```diff
@@ -93,12 +87,8 @@ def __init__(
                                 the source version)
           --root_dir ROOT_DIR   Root directory for the collections project.
                                 (default: /global/cfs/cdirs/kbase/collections)
-          --threads THREADS     Total number of threads used by the script. (default:
-                                half of system cpu count)
-          --program_threads PROGRAM_THREADS
+          --threads_per_tool_run THREADS_PER_TOOL_RUN
```

Comment:
This seems like it won't work with the current task generation code, aren't changes needed there?

Reply:
yea. I will need to update taskfarmer task generation code and update entrypoint script for each tool container in order of this to work. But since we didn't update containers yet, it won't break anything at least.

Reply:
Doesn't changes in tool common cause the images to get built? I don't really remember how that works

Reply:
no. Only when we update the yaml file for each tool.

Reply:
:+1: